src/components/Comments.jsx

```diff
@@ -14,15 +22,50 @@ function Comments({ article_id }) {
         });
     }, [article_id]);
 
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        console.log(newComment, user)
+        postComment(article_id, { body: newComment, username: user.username })
+            .then((response) => {
+                setComments([...comments, response]);
```

Comment:
Would be better for the new comment to appear at the top

Reply:
solved